src/OrchardCore/OrchardCore.Users.Abstractions/Handlers/IUserEventHandler.cs

```diff
@@ -0,0 +1,13 @@
+using System.Threading.Tasks;
+
+namespace OrchardCore.Users.Handlers
+{
+    public interface IUserEventHandler
```

Comment:
@MatthijsKrempel Please see this as I think you did something different in your PR to for user activation.

Reply:
I created a seperate interface, but this also good.